Sources/JsonModel/ResultData/FileResult.swift

```diff
@@ -2,7 +2,7 @@
 //  FileResultObject.swift
 //  
 //
-//  Copyright © 2020 Sage Bionetworks. All rights reserved.
+//  Copyright © 2017-2021 Sage Bionetworks. All rights reserved.
```

Comment:
2017 or 2020 (previous copyright)?

Reply:
This file now includes the *protocol* as well as an object that conforms to that protocol. The protocol was written in 2017.